frontend/readhistory.lua

```diff
@@ -199,6 +200,22 @@ function ReadHistory:updateItemByPath(old_path, new_path)
     end
 end
 
+--- Updates the history list after renaming/moving a folder.
+function ReadHistory:updateItemsByPath(old_path, new_path)
+    old_path = "^"..old_path
```

Comment:
Oof, this was so confusing to me. I think it'd be much clearer straight in the gsub, heh. :-)

Reply:
> straight in the gsub

It's in the loop.

Reply:
Or old_path_pattern instead of just old_path